pkg/cache/cache.go

```diff
@@ -263,6 +341,9 @@ func (c *ClusterQueue) addWorkload(w *kueue.Workload) error {
 	wi := workload.NewInfo(w)
 	c.Workloads[k] = wi
 	c.updateWorkloadUsage(wi, 1)
+	if !apimeta.IsStatusConditionTrue(w.Status.Conditions, kueue.WorkloadPodsReady) {
```

Comment:
also, if `c.podsReadyTracking`

Reply:
we are here in cluster queue, so also needed to pass the `podsReadyTracking` down from cache.